### ISSUE TITLE
SK4 v4.2.2 crashes on SSP world-load ; `IllegalStateException: failed to create a child event loop`

### ISSUE THREAD

**Issue by ASchneider-GitHub** [DarkPacks/SkyFactory-4#2000]
## Bug Description

 Attempting to launch a singleplayer world on a clean installation immediately crashes the game

### Did this behavior use to work in the previous version?

I haven't played a previous version

## Steps to Reproduce (for bugs)

1. Download clean installation of SK4 v4.2.2
2. Launch game and create new SSP world with default settings
3. Game crashes while loading

## Logs

* Client/Server Log:
* Crash Log: https://pastebin.com/sjeLMQUq

## World Information
* World Preset: Sky Factory 4
* Prestige Enabled: Disabled
* Modpack Version world created in: v4.2.2
* Additional Content Installed: N/A

## Client Information
* Modpack Version: 4.2.2
* Java Version: 16.0.1.0 (64-Bit)
* Launcher Used: CurseForge
* Memory Allocated: 8GB
* Server/LAN/Single Player: Singleplayer (no LAN)
* Optifine Installed: No
* Shaders Enabled: No

**Comment by sam-kirby** [comment]
This looks to be a vanilla Minecraft issue: https://bugs.mojang.com/browse/MC-34749

Either your security software is causing the crash, or Minecraft is unable to establish an internet connection over IPv6.

**Comment by ASchneider-GitHub** [comment]
Makes sense. The PC runs over an IPv4 VPN tunnel, so IPv6 is likely blocked. Guess I'll try forcing it to run on IPv4 with `-Djava.net.preferIPv4Stack=true` or disabling the VPN

**Comment by sam-kirby** [comment]
You could also try turning off DHCPv6 on your router if possible. It won't make an attempt to connect over IPv6 if your system does not have a v6 address.
